config-templates/shopify/order.yaml

```diff
@@ -0,0 +1,59 @@
+data_source:
+  name: shopify_orders
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY."ORDER" ### Update to your opportunities schema.table
+  mutability:
+    type: immutable
+  identifiers:
+    - name: order
+      type: primary
+      expr: id
+    - name: shipping_address_id
```

Comment:
most of the identifiers in this model exclude `_id` - for consistency, remove here (i think it's easier to query without the `_id`, too